stable/fluent-bit/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 1.9.2
+version: 1.9.3
```

Comment:
Its a new feature, so should be 1.10.0. Otherwise, looks good to me.

Reply:
I'm agree with you.